RecoJets/JetProducers/python/ecfAdder_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+ECF = cms.EDProducer("ECFAdder",
```

Comment:
some longer and more intuitive name would be better.
Our rules are to name the module instance and the cfi file with the same name.
I like ecfAdder more than ECF (even though ecf could've been more descriptive).